src/server/pegasus_server_impl.cpp

```diff
@@ -3375,12 +3389,17 @@ uint64_t pegasus_server_impl::do_manual_compact(const rocksdb::CompactRangeOptio
     // update rocksdb statistics immediately
     update_replica_rocksdb_statistics();
 
-    uint64_t last_manual_compact_finish_time = 0;
-    CHECK_OK_PREFIX(
-        _meta_store->get_last_manual_compact_finish_time(&last_manual_compact_finish_time));
     return last_manual_compact_finish_time;
 }
 
+void pegasus_server_impl::after_manual_compact(std::uint64_t starttime, uint64_t endtime)
+{
+    // store last manual compact used time to meta store for learn situation
```

Comment:
If the replica server shutdown before the replicas complete the manual compaction, can this patch resolve this issue?